docs/DeveloperGuide.md

```diff
@@ -1,34 +1,614 @@
+object, where the information of the appointment is again stored in a Map that the ```AddAppoinmentCommand``` object would read from. 
+Said information will be stored in the ```AddAppoinmentCommand``` object
+
+2. When 
+the ```Main``` calls ```execute(Ui ui, Storage storage)```, the ```AddAppointmentCommand``` class would call upon the 
+```Appointment``` class to make an ```Appointment``` Object. 
+
+3. After which, the ```AddAppoinmentCommand``` object will 
+call upon the ```AppointmentList``` object to obtain the list of ```Appointments``` (get the ```List``` object that 
+represents the list of appointments by ```AppointmentList```’s ```getAppointmentList()``` command) so that it can 
+directly add the new ```Appointment``` object into the appointment list. 
+
+4. Finally, it will call upon the ```Storage``` 
+class’s ```saveAppoinmentList()``` function to save the updated appointment list. 
+
+5. Upon successfully adding the 
+```Appointment``` object into the appointment list, it will call upon the ```Ui``` class’ 
+```showAppointmentAddSuccess()``` function to display the success of adding the ```appointment``` into the appointment 
+list.
+
+Below shows the sequence diagram for ```AddAppointmentCommand``` class
+
+![](images/AddAppointCommandSequence.png)
+
+##### 2.2.4.3 ListPatientClass
+
+To display the list of patients, the ```ListPatientCommand``` class is called. This class serves as a façade class of 
+```Main``` and ```Ui``` to interact with each other. 
+
+![](images/ListPatientDiagram.png)
+
+1. This class' object is first created by the ```Parser``` class, where it is then returned to the ```Main``` class to have its
+```execute(Ui ui, Storage storage)``` function be called. 
+
+2. When the ```Main``` class calls the 
+```execute(Ui ui, Storage storage)``` function, ```ListPatientCommand``` will call upon the ```Ui```’s 
+```showEntirePatientList()``` function to display the list of patients.
+
+Below shows the sequence diagram for ```ListPatientCommand``` class
+
+![](images/ListPatientCommandSequence.png)
+
+##### 2.2.4.4 ListAppointmentClass
+
+To display the list of appointments, the ```ListAppointmentCommand``` class is called. This class serves as a façade 
+class of ```Main``` and ```Ui``` to interact with each other. 
+
+![](images/ListAppointmentDiagram.png)
+
+1. This class' object is first created by the ```Parser``` class, where it is then returned to the ```Main``` class to 
+have its ```execute(Ui ui, Storage storage)``` function be called. 
+
+2. When the ```Main``` class calls the 
+```execute(Ui ui, Storage storage)``` function, ```ListAppointmentCommand``` will call upon the ```Ui```’s 
+```showEntireAppointmentList()``` function to display the list of appointments.
+
+Below shows the sequence diagram for ```ListAppointmentCommand``` class
+
+![](images/ListAppointmentCommandSequence.png)
```

Comment:
![image](https://user-images.githubusercontent.com/31772358/77994001-d6bab580-735b-11ea-9852-0a62b65fb8ad.png)

Same here: should there be a cut at the lower right corner of the loop rectangle?